src/antsibull_docs/schemas/docs/ansible_doc.py

```diff
@@ -78,64 +81,54 @@ class AnsibleDocSchema(BaseModel):
     role: dict[str, RoleSchema]
 
 
-class GenericPluginSchema(BaseModel):
-    """
-    Document the output of ``ansible-doc -t PLUGIN_TYPE PLUGIN_NAME``.
-
-    .. note:: Both the model and the dict will be wrapped in an outer dict with your data mapped
-        to the ``__root__`` key. This happens because the toplevel key of ansible-doc's output is
-        a dynamic key which we can't automatically map to an attribute name.
-    """
-
-    __root__: dict[str, PluginSchema]
-
-
-class CallbackPluginSchema(BaseModel):
-    """
-    Document the output of ``ansible-doc -t callback CALLBACK_NAME``.
-
-    .. note:: Both the model and the dict will be wrapped in an outer dict with your data mapped
-        to the ``__root__`` key. This happens because the toplevel key of ansible-doc's output is
-        a dynamic key which we can't automatically map to an attribute name.
-    """
+GenericPluginSchema = p.RootModel[dict[str, PluginSchema]]
+"""
+Document the output of ``ansible-doc -t PLUGIN_TYPE PLUGIN_NAME``.
 
-    __root__: dict[str, CallbackSchema]
+.. note:: Both the model and the dict will be wrapped in an outer dict with your data mapped
+    to the ``__root__`` key. This happens because the toplevel key of ansible-doc's output is
+    a dynamic key which we can't automatically map to an attribute name.
```

Comment:
Is this still the case with the way new `RootModel`s work?

Reply:
No, I forgot to update the comments. I removed the notes in ff7148607b96bcc0046e8f308d0d09e963c58ae0.